cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinHrcsCategoriesAndActiveties.java

```diff
@@ -0,0 +1,52 @@
+package no.unit.nva.cristin.mapper;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import nva.commons.core.JacocoGenerated;
+
+@Data
+@Builder(
+        builderClassName = "CristinHrcsBuilder",
+        toBuilder = true,
+        builderMethodName = "builder",
+        buildMethodName = "build",
+        setterPrefix = "with"
+)
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class CristinHrcsCategoriesAndActiveties {
+
+    private static final String CATEGORY = "helsekategorikode";
+    private static final String ACTIVITY = "aktivitetskode";
+
+    public static final String HRCS_CATEGORY_URI = "https://nva.unit.no/hrcs/category/";
```

Comment:
This is not going to work for any other environment except for production. We have done similar work in the past, use the env variables for the custom domain.